### ISSUE TITLE
Cargo.lock is missing

### ISSUE THREAD

**Issue by johnae** [greshake/i3status-rust#608]
I couldn't find any other issue filed on this but builds started failing for me in CI since the lock file is missing from the repo. Is this intentional?

**Comment by johnae** [comment]
Yeah, this pull request seems to have deleted the lock file: https://github.com/greshake/i3status-rust/pull/558

**Comment by ammgws** [comment]
Sorry about that
